src/SlateAsInputEditor/index.test.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import Enzyme, { shallow } from "enzyme";
+import SlateAsInputEditor from "./index";
+import Adapter from "enzyme-adapter-react-16";
+
+import NoEdit from '../../src/plugins/noedit';
+import List from '../../src/plugins/list';
+import { SlateTransformer } from '@accordproject/markdown-slate';
+
+const plugins = [NoEdit(), List()];
+
+Enzyme.configure({adapter: new Adapter()});
+
+// Default markdown value
+let defaultValue = `My Heading
+
+This is text. This is *italic* text. This is **bold** text. This is a [link](https://clause.io). This is \`inline code\`.
+
+This is ***bold and italic*** text
+
+> This is a quote.
+
+Heading Two
+
+This is more text.
+
+Ordered lists:
+
+1. one
+2. two
+3. three
+
+Or:
+
+- apples
+- pears
+- peaches
+
+### Sub heading
+
+This is more text.
+
+Fin.`;
+
+const slateTransformer = new SlateTransformer();
+const propsObj = { width: '600px'};
+
+const onSlateValueChange = () => {
+    // A dummy onChange props
+  };
+
+describe("SlateAsInputEditor component", () => {
+    test("component mount", () => {
```

Comment:
I think this test could be split into two maybe. There's the `component mounts` which is line 59-60. Then there is a `successful roundtrip` which is line 54-57.

Reply:
@irmerk Done, I have split the test cases into 2